packages/cli/src/util/validate-paths.ts

```diff
@@ -78,14 +74,12 @@ export default async function validatePaths(
     return { valid: false, exitCode: 1 };
   }
 
-  const isFile = pathStat && !pathStat.isDirectory();
-  if (isFile) {
-    output.print(
-      `${prependEmoji(
-        'Deploying files with Vercel is deprecated (https://vercel.link/faq-deploy-file)',
-        emoji('warning')
-      )}\n`
-    );
+  if (!pathStat.isDirectory()) {
+    output.prettyError({
+      message: 'Support for single file deployments has been removed.',
+      link: 'https://err.sh/vercel/no-single-file-deployments',
```

Comment:
I don't think we use err.sh anymore. This should be vercel.link

Reply:
K, done. Right now it's just pointing to the error markdown page on this PR, like how err.sh would have done it. Let me know if you think we should link to somewhere else.